cli/src/main/java/com/devonfw/tools/ide/tool/dotnet/DotNet.java

```diff
@@ -0,0 +1,37 @@
+package com.devonfw.tools.ide.tool.dotnet;
+
+import com.devonfw.tools.ide.common.Tag;
+import com.devonfw.tools.ide.context.AbstractIdeContext;
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.process.ProcessMode;
+import com.devonfw.tools.ide.tool.LocalToolCommandlet;
+
+import java.util.Set;
+
+public class DotNet extends LocalToolCommandlet {
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}. method.
+   */
+  public DotNet(IdeContext context) {
+
+    super(context, "dotnet", Set.of(Tag.DOTNET, Tag.CS));
+  }
+
+  @Override
+  public void run() {
+
+    AbstractIdeContext abstractIdeContext = (AbstractIdeContext) context;
+    abstractIdeContext.setDefaultExecutionDirectory(context.getIdeHome());

```

Comment:
We should not change the working directory. For the end-user calling `dotnet «args»` vs. `ide dotnet «args»` should cause the same behavior (except that calling via `ide` will ensure that dotnet gets first installed and put on the PATH if not already done) and IDEasy should not get in the way here.
```suggestion

```